scripts/model_gen_ng/model_gen_ng/struct.py

```diff
@@ -0,0 +1,188 @@
+import re
```

Comment:
Copyright notice as all the Python files.
If we don't have it, please add it to the others too.